hadoop-yarn-project/hadoop-yarn/hadoop-yarn-common/src/main/resources/yarn-default.xml

```diff
@@ -715,6 +715,14 @@
     <value>org.apache.hadoop.yarn.client.ConfiguredRMFailoverProxyProvider</value>
   </property>
 
+  <property>
+    <description>When HA is not enabled, the class to be used by Clients, AMs and
```

Comment:
Is there any other place where we can document this proxy? Maybe one of the HA md files.

Reply:
Added documentation